docs/generated/packages/js/generators/library.json

```diff
@@ -58,7 +61,8 @@
       },
       "importPath": {
         "type": "string",
-        "description": "The library name used to import it, like @myorg/my-awesome-lib."
+        "description": "The library name used to import it, like @myorg/my-awesome-lib.",
+        "x-priority": "important"
```

Comment:
Since there are good defaults generated for this, I think most people won't update this flag manually. So imo it shouldn't be important. What do you think?